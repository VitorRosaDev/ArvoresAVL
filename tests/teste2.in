i 50
i 30
i 70
i 65
b 65
b 10
sair