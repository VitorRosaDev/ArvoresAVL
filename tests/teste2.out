OK: inserido 50
Árvore:
└── 50
OK: inserido 30
Árvore:
└── 50
    └── 30
OK: inserido 70
Árvore:
└── 50
    ├── 30
    └── 70
OK: inserido 65
Árvore:
└── 50
    ├── 30
    └── 70
        └── 65
Caminho: 50 70 65
Encontrado: sim
Caminho: 50 30
Encontrado: não
Encerrado.